examples/step-42/doc/intro.dox

```diff
@@ -321,7 +321,7 @@ method for the contact. It works as follows:
  Here,
  $P_{\mathcal{A}}(U)$ is the projection of the active
  components in $\mathcal{A}$ to the gap
- @f{gather*}P_{\mathcal{A}}(U)_p \dealcoloneq \begin{cases}{
+ @f{gather*}P_{\mathcal{A}}(U)_p \dealcoloneq \begin{cases}
```

Comment:
I bet these opening braces were introduced by a script and were meant for the `@f{gather}` at the top of the line.